test/e2e/network/util_iperf.go

```diff
@@ -46,6 +52,10 @@ type IPerfResult struct {
 	bandwidthBits int64
 }
 
+func (i *IPerfResult) bandwidthMB() int {
+	return int(math.Round(float64(i.bandwidthBits) / float64(megabyte) / 8))
+}
```

Comment:
the IPerfResult type is using int64, this return an `int` and you are converting it to a float64 :smile:

Reply:
I wish there were a way to just round to an int in golang!!

Reply:
why not using int64? int depends on the platform, it can be int32 or int64

Reply:
oh got it!  💯